PackagePrice.java

```diff
@@ -0,0 +1,52 @@
+package pricecomparison;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+public class PackagePrice {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		WebDriver driver = new FirefoxDriver();
+
+		String appUrl = "http://www.tripfactory.com/package/get-price-req?pkgId=27433&ahdate=06%2F05%2F2016";
+		driver.get(appUrl);
+
+		WebElement wE = driver.findElement(By.xpath(".//*[@id='invCalMth2']/table"));
+
+		System.out.println(wE.getText());
+
+		driver.findElement(By.xpath(".//*[@id='invCalMth2']/table/tbody/tr[2]/td[1]/div[2]/div[3]")).click();

```

Comment:
17th line is used to get the price present on calendar dates(1 person).and 21st line is used to select a particular date to get its booking summary price (for 2 persons).